src/common/mod.rs

```diff
@@ -11,6 +11,45 @@ pub(crate) mod system;
 #[cfg(feature = "user")]
 pub(crate) mod user;
 
+/// Type containing read and written bytes.
+///
+/// It is returned by [`Process::disk_usage`][crate::Process::disk_usage] and [`Disk::usage`][crate::Disk::usage].
+///
+/// ```no_run
+/// use sysinfo::{Disk, System};
+///
+/// let s = System::new_all();
+/// for (pid, process) in s.processes() {
+///     let disk_usage = process.disk_usage();
+///     println!("[{}] read bytes   : new/total => {}/{} B",
+///         pid,
+///         disk_usage.read_bytes,
+///         disk_usage.total_read_bytes,
+///     );
+///     println!("[{}] written bytes: new/total => {}/{} B",
+///         pid,
+///         disk_usage.written_bytes,
+///         disk_usage.total_written_bytes,
+///     );
+/// }
+///
+/// let disks = Disks::new_with_refreshed_list();
+/// for disk in disks.list() {
+///     println!("[{:?}] disk usage: {:?}", disk.name(), disk.usage());
+/// }
+/// ```
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq, PartialOrd)]
+pub struct DiskUsage {
```

Comment:
Not sure if moving `DiskUsage` here is the right move

Reply:
Not sure why you moved it either. ^^'

Reply:
🤦‍♂️ I should have provided context.

I decided to move it because of compilation errors with `cargo check --no-default-features --features=debug,serde,disk`. Since `DiskUsage` was previously in `system.rs` gated behind the `system` feature, the check command fails.

We could include `DiskUsage` as part of the `disk` feature, but `system` feature would then require `disk` as well, and I am not sure we want that.

Reply:
Ah right! Well makes sense. Please add `cfg(any(feature = "disk", feature = "system"))` on it. But otherwise, good call moving it here. It was the right move.